pyproject.toml

```diff
@@ -83,6 +82,16 @@ dependencies = [
     "typing-extensions~=4.10; python_version < '3.13'",
 ]
 
+# Static URLs for GitHub dependency graph compatibility
+# See https://github.com/collective/icalendar/issues/1035
+[project.urls]
+Homepage = "https://icalendar.readthedocs.io/en/stable/"
+Repository = "https://github.com/collective/icalendar"
+Issues = "https://github.com/collective/icalendar/issues"
+Documentation = "https://icalendar.readthedocs.io/en/stable/"
```

Comment:
```suggestion
Documentation = "https://icalendar.readthedocs.io/"
```

Reply:
@niccokunzmann but we want them to go directly to the stable version, not redirect to it from the naked hostname, hence the `/en/stable/`.